### DISCUSSION TITLE
Web application for YASA

### DISCUSSION THREAD

**Discussion by bartromb** [raphaelvallat/yasa#185] [General]
Hi everyone,

I’ve developed a web application that integrates the excellent [yasa](https://raphaelvallat.com/yasa/) library into a user-friendly web platform. The application, named YASA Flaskified, is built with Flask, Gunicorn, and Nginx. I currently use it on internal servers as a quality control step after manual scoring. (I am a pulmonologist and somnologist.)

You can find it on my GitHub repository: [YASA Flaskified](https://github.com/bartromb).

I have used only Somnomedics EDF files so far, but the script attempts to automatically identify EEG channels, so I believe it is usable for other EDF files as well. It generates both a hypnogram PDF and a CSV output for further analysis. You can see a showcase on [sleepai.be](http://sleepai.be/) and [sleepai.eu](http://sleepai.eu/), though due to bandwidth constraints, only admin users can register new accounts.  I can provide a test login and password upon request.

I want to thank Raphael for the fantastic [yasa](https://raphaelvallat.com/yasa/) library—it has been incredibly useful for this project!
Contributions and feedback are more than welcome, and I’m happy to assist with any issues you encounter during deployment on a server. 

Looking forward to hearing your thoughts!

Best regards,
Bart

**Comment by remrama** [comment]
Closing this Discussion as it moved to #186
